app/views/posts/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<div>
+  <h1>Create a new post:</h1>
+  <%= form_with model: @post, url: user_posts_path, method: :post do |form| %>
+    <%= form.label :title, 'Title' %>
+    <%= form.text_field :title %>
+    <%= form.label :text, 'Text' %>
+    <%= form.text_area :text %>
+    <%= form.submit 'Post' %>
+  <% end %>
```

Comment:
- Okay 👍 
![Capture](https://user-images.githubusercontent.com/63972670/215128619-fafddfc8-7bd1-49bf-8665-a10bbd551d88.JPG)

Reply:
Hi @OLIPLICHE  
thanks for your review